cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -119,6 +121,9 @@ public class CruiseControlTest {
     private final int healthDelay = 120;
     private final int healthTimeout = 30;
     private final String minInsyncReplicas = "2";
+    static String brokerCapacityCpu = "6.0";
+    static String brokerCapacityOverrideCpu = "2.0";
+    static String resourceLimitCpu = "3.0";
```

Comment:
Is there any reason why are these not `private static final`? I guess the same question applies also to the fields above this.

Reply:
All have been refactored to `private static final`